packages/ui/v2/core/form/fields.tsx

```diff
@@ -453,6 +453,9 @@ export function InputGroupBox(props: JSX.IntrinsicElements["div"]) {
   );
 }
 
-export const MinutesField = forwardRef<HTMLInputElement, InputFieldProps>(function MinutesField(props, ref) {
-  return <InputField ref={ref} type="number" min={0} {...props} addOnSuffix="mins" />;
+export const DurationField = forwardRef<HTMLInputElement, InputFieldProps>(function DurationField(
```

Comment:
I think there is very little reason to keep this component in here now, since it's actually just an input field with `type=number`. What do you think about removing it here and just use the `InputField` on the page?